packages/fuel-indexer-api-server/src/ffi.rs

```diff
@@ -0,0 +1,75 @@
+use wasmer::{
+    imports, AsStoreMut, Exports, Function, Instance, MemoryView, StoreMut, WasmPtr,
+};
+
+/// Extract the `TOOLCHAIN_VERSION` string from a WASM module. This function
+/// creates a `wasmer::Instance` in order to do this.
+pub(crate) fn check_wasm_toolchain_version(data: Vec<u8>) -> anyhow::Result<String> {
+    let mut store = wasmer::Store::default();
+
+    let module = wasmer::Module::new(&store, data.clone())?;
+
+    let mut exports = Exports::new();
+    exports.insert(
```

Comment:
I guess `wasmer` requires us to add these exports even though they're not actually used?